windows/webview.cc

```diff
@@ -22,6 +22,15 @@ inline void ConvertColor(COREWEBVIEW2_COLOR& webview_color, int32_t color) {
   webview_color.A = (color >> 24) & 0xFF;
 }
 
+std::wstring get_utf16(const std::string &str, int codepage)

```

Comment:
This should be formatted according to the `.clang-format` file in the project root.

Reply:
Sorry, i don't understand ... 

In .clang-format i only found this directives

![image](https://user-images.githubusercontent.com/27212546/147824952-a1a13e1a-b96f-440b-a68e-aa799c09efb9.png)

Can you explain a bit more?

Reply:
Sorry for this... fixed now...